src/components/stake/StakeAmountSlider.vue

```diff
@@ -0,0 +1,577 @@
+                        <OneLeafStakingIcon v-if="stakePercentage.value < 50" />
+                        <StakingIcon v-else-if="stakePercentage.value < 75" />
+                        <ThreeLeafStakingIcon v-else />
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Ref, defineComponent, ref, computed, onMounted } from '@vue/composition-api';
+import { Amount } from '@nimiq/vue-components';
+import { useAddressStore } from '../../stores/Address';
+import { calculateDisplayedDecimals, formatSpaceyNumber } from '../../lib/NumberFormatting';
+
+import StakingIcon from '../icons/Staking/StakingIcon.vue';
+import OneLeafStakingIcon from '../icons/Staking/OneLeafStakingIcon.vue';
+import ThreeLeafStakingIcon from '../icons/Staking/ThreeLeafStakingIcon.vue';
+import VerticalLineIcon from '../icons/Staking/VerticalLineIcon.vue';
+
+import { CryptoCurrency, NIM_DECIMALS, NIM_MAGNITUDE } from '../../lib/Constants';
+
+const getSVGNode = (n:string, attrs:Record<string, string> = {}) => {
+    const e = document.createElementNS('http://www.w3.org/2000/svg', n);
+    Object.entries(attrs).forEach((entry) => {
+        e.setAttributeNS(null, entry[0], entry[1]);
+    });
+    return e;
+};
+
+const encodedSVG = (element:SVGElement) => {
+    const serializedSVG = new XMLSerializer().serializeToString(element);
+    const base64Data = window.btoa(serializedSVG);
+    return `data:image/svg+xml;base64,${base64Data}`;
+};
+
+const buildSVG = (nodes:Array<Array<any>> = [], encoded = true) => {
+    const svg = getSVGNode('svg');
+    for (const attr of nodes) svg.appendChild(getSVGNode(attr[0] as string, attr[1] as Record<string, string>));
+    return encoded ? encodedSVG(svg) : svg;
+};
+
+type Point = {
+    x: number,
+    y: number,
+} | null
+
+const extractInfo = (e: MouseEvent | TouchEvent):Point => {
+    if (e.type === 'touchstart') {
+        e = e as TouchEvent;
+        if (e.touches.length > 1) return null;
+        const t = e.touches[0];
+
+        return {
+            x: t.pageX,
+            y: t.pageY,
+        };
+    }
+
+    e = e as MouseEvent;
+    return {
+        x: e.pageX,
+        y: e.pageY,
+    };
+};
+
+export default defineComponent({
+    setup(props, context) {
+        const { activeAddressInfo } = useAddressStore();
+
+        const readonly = ref(false);
+        const active = ref(false);
+        const alreadyStakedAmount = ref(props.stakedAmount);
+        const currentAmount = ref(alreadyStakedAmount.value);
+        const availableAmount = ref(activeAddressInfo.value?.balance);
+        const currentPercentage = computed({
+            get: () => (100 * currentAmount.value) / availableAmount.value!,
+            set: () => {
+                //
+            },
+        });
+        const alreadyStakedPercentage = ref(currentPercentage.value);
+        const alreadyStaked = ref(alreadyStakedAmount.value > 0);
```

Comment:
`alreadyStaked` and `alreadyStakedPercentage` are not reactive to changes to `alreadyStakedAmount` and should be computed properties instead.